modules/ROOT/pages/mel-dataweave-functions.adoc

```diff
@@ -76,7 +76,7 @@ See xref:dataweave-formats.adoc[DataWeave Formats] for a reference of all the di
 
 == The Split Function
 
-The `split()` function runs an expression written in DataWeave language and returns an iterator who's result can then be processed by your flow one index at a time. It takes at least two arguments: a DataWeave expression and the desired output type. This function may be equivalent to running the 'dw()' function and then adding an additional step to your flow, such as a xref:foreach.adoc[Foreach Scope], but it's less heavy on resources.
+The `split()` function runs an expression written in DataWeave language and returns an iterator the result of which can then be processed by your flow one index at a time. It takes at least two arguments: a DataWeave expression and the desired output type. This function may be equivalent to running the 'dw()' function and then adding an additional step to your flow, such as a xref:foreach.adoc[Foreach Scope], but it's less heavy on resources.
```

Comment:
Maybe split the long sentence in two?

 "The `split()` function runs an expression written in DataWeave language and returns an iterator. Your flow can process the result of the iterator one index at a time. The iterator takes at least two arguments...

Change "may be" to "can be." "May" implies permission.